src/controllers/routes.ts

```diff
@@ -1,12 +1,12 @@
-import { Router } from "@well-known-components/http-server"
-import { GlobalContext } from "../types"
-import { pingHandler } from "./handlers/ping-handler"
+import { Router } from '@well-known-components/http-server'
+import { GlobalContext } from '../types'
+import { resizeHandler } from './handlers/resize-handler'
 
 // We return the entire router because it will be easier to test than a whole server
 export async function setupRouter(globalContext: GlobalContext): Promise<Router<GlobalContext>> {
   const router = new Router<GlobalContext>()
 
-  router.get("/ping", pingHandler)
+  router.get('/content/:hash/dxt/:length', resizeHandler)
```

Comment:
nitpick: maybe this should be a POST request? (feel free to ignore)

Reply:
I can change it but what would be the reason of having this endpoint as a POST?

Reply:
I think in this case we want it to be a GET, because the explorer won't post to "request" the converted asset. Also POST are not cacheable by the cdn and edge servers